build.sbt

```diff
@@ -16,32 +16,40 @@
 import sbt.Keys._
 import sbt._
 import com.twitter.sbt._
+import sbt.Defaults._
 
 lazy val Versions = new {
   val logback = "1.2.1"
+  val sbt = "0.13.11"
   val util = "0.30.1"
   val json4s = "3.5.0"
   val datastax = "3.1.0"
   val scalatest = "3.0.0"
   val shapeless = "2.3.2"
   val thrift = "0.8.0"
-  val finagle = "6.37.0"
+  val finagle = "6.42.0"
   val scalameter = "0.8+"
   val scalacheck = "1.13.4"
   val slf4j = "1.7.21"
   val reactivestreams = "1.0.0"
   val cassandraUnit = "3.0.0.1"
   val javaxServlet = "3.0.1"
-  val typesafeConfig = "1.3.1"
+
   val joda = "2.9.7"
   val jodaConvert = "1.8.1"
   val scalamock = "3.4.2"
   val macrocompat = "1.1.1"
   val macroParadise = "2.1.0"
 
+  val typesafeConfig: String = if (Publishing.isJdk8) {
```

Comment:
@iRhysBradbury Obviously because `1.3.1` is not JDK7 bytecode compatible because it explicitly references stuff in the JDK 8 standard lib.